adapter_nina.go

```diff
@@ -0,0 +1,113 @@
+//go:build nina || nano_rp2040
```

Comment:
Would it be possible to use just a single build tag for this? For example by adding the `nina` build tag to targets/nano-rp2040.json.

(Also, I wonder whether `nina` is perhaps a bit too generic? Not sure how likely it is that it will be used in a place that is not actually that specific BLE chip).

Reply:
I have changed it to `ninafw` so that it is closer to the name of the firmware https://github.com/arduino/nina-fw